content/docs/reference/addon-api/addon.tab/addon.tab.traps.md

```diff
@@ -12,8 +12,6 @@ aliases:
 
 `addon.tab.traps` is a public API for userscripts that can be used to obtain objects that are not otherwise available. The actual traps are coded at [prototype-handler](https://github.com/ScratchAddons/ScratchAddons/blob/master/content-scripts/prototype-handler.js).
 
-They share some APIs with "Once" traps, but is very different and is documented at [Fake redux state](Fake-redux-state) separately.  

```

Comment:
@WorldLanguages Do you know what this was referring to?

Reply:
I checked the git history and apparently there was never a page with this name.

Reply:
This line is good to remove then

Reply:
I look the old wiki and it seems that it is removed on v1.3.0. So, this is irrelevant and may be removed.